spec/repositories/valkyrie/file_repository_spec.rb

```diff
@@ -2,7 +2,7 @@
 require 'rails_helper'
 
 RSpec.describe Valkyrie::FileRepository do
-  let(:repository) { instance_double(Valkyrie::FileRepository::Memory) }
+  let(:repository) { Valkyrie::FileRepository::Memory.new }
```

Comment:
It's actually because registering an instance_double as a repository was making the tests fail when run in certain orders. :(